### DISCUSSION TITLE
Interesting aircraft on map

### DISCUSSION THREAD

**Discussion by aa0pb** [vradarserver/vrs#47] [Ideas]
Howdy,
Is it possible (now or as a future item) to have flagged "interesting" aircraft show as a different color (like red or something) on the map?  I know it is shown on the list at the top, but it would be nice to have a visual reference on the map on where they are.

Thanks, and great app,
Phil

**Comment by vradarserver** [comment]
It's possible to do it via the custom content plugin but only if you're using the version 3 preview and you're comfortable writing JavaScript. An example of a script that dynamically changes the fill colour of a marker is here - https://github.com/vradarserver/vrs-custom-scripts/tree/master/Change%20V3%20SVG%20Marker%20Fill%20Per%20Criteria.
